containers-tests/tests/intmap-strictness.hs

```diff
@@ -76,6 +82,70 @@ pInsertLookupWithKeyValueStrict f k v m
                      not (isBottom $ M.insertLookupWithKey (const3 1) k bottom m)
     | otherwise    = isBottom $ M.insertLookupWithKey (apply3 f) k bottom m
 
+pFromAscListLazy :: [Int] -> Bool
+pFromAscListLazy ks = not . isBottom $ M.fromAscList elems
+  where
+    ks' = map head . group . sort $ ks
```

Comment:
Why not just use `Data.Containers.ListUtils.nubInt`? And why do the elements have to be unique at all? We're not testing `fromDistinctAscList` here.

Reply:
Duplicate keys would actually invalidate this check; `fromAscList [(0,undefined),(0,()),(0,undefined),(0,())]` is bottom. (Not saying that this is how it should be... but that's how it is and has been for a long time.) 

I was not actively aware of `nubInt`, somehow.

Anyway I've addressed this and the other comments as well; I found a few minor discrepancies on top (like `Map` tests having `IntMap` in their name...)